package.json

```diff
@@ -27,6 +27,7 @@
     "react-redux": "^5.0.7",
     "react-router-dom": "^4.2.2",
     "react-scripts-ts": "2.14.0",
+    "react-vis-force": "^0.3.1",
```

Comment:
Mrrrr does not say it enables tree shaking at https://bundlephobia.com/result?p=react-vis-force@0.3.1. Would be great for us to separate out things like this a bit so it will only get loaded for this one page instead for all.